node-manager/src/node.rs

```diff
@@ -166,18 +175,65 @@ pub(crate) async fn connect_peer_if_necessary(
     */
 
     // first make a connection to the node
-    let tcp_proxy = TcpProxy::new(websocket_proxy_addr, peer_addr).await;
-
-    // TODO remove the test send
-    tcp_proxy.send(String::from("test\n").into_bytes().to_vec());
+    let tcp_proxy = Arc::new(TcpProxy::new(websocket_proxy_addr, peer_addr).await);
+    let mut descriptor = SocketDescriptor::new(tcp_proxy);
 
     // TODO then give that connection to the peer manager
+    let initial_bytes = peer_manager.new_outbound_connection(
+        pubkey,
+        descriptor.clone(),
+        Some(get_net_addr_from_socket(peer_addr)),
+    )?;
+
+    let sent_bytes = descriptor.send_data(&initial_bytes, true);
+    debug!("sent some bytes to a node: {sent_bytes}");
```

Comment:
nit: since `sent_bytes` is a number would read better as `sent {sent_bytes} to node: {pubkey}`

Reply:
added